src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1299,5 +1306,115 @@ public Task CompactDatabase()
                 NoContentStatus();
             }
         }
+
+        [RavenAction("/admin/offline-migrate", "POST", AuthorizationStatus.ClusterAdmin)]
```

Comment:
@talweiss1982 Can you rename to /admin/migrate/offline to be consistent with other migration endpoints?

Reply:
sure